Doppler.HtmlEditorApi/Infrastructure/Repository.cs

```diff
@@ -0,0 +1,180 @@
+using System;
+using Doppler.HtmlEditorApi.Model;
+using System.Threading.Tasks;
+
+
+namespace Doppler.HtmlEditorApi.Infrastructure
+{
+    public class Repository : IRepository
```

Comment:
In this case, it is fine to use a more specific name. It is _the dummy implementation_ of the _repository abstraction_

```suggestion
    public class DummyRepository : IRepository
```

Reply:
Fixed